toolkit/javascripts/analytics/_register.js

```diff
@@ -12,6 +12,20 @@
         universalId: universalId,
         cookieDomain: cookieDomain
       });
+      /* Add GOV.UK cross domain tracking
+       Calls the equivalent commands in node_modules/govuk_frontend_toolkit/javascripts/govuk/analytics/google-analytics-universal-tracker.js (need to check if this format is OK):
```

Comment:
_is_ the format OK?

Reply:
Haha yes it is! (I checked with Andy.) I can remove that comment now 👍

Reply:
🍾